posthog/tasks/calculate_cohort.py

```diff
@@ -123,6 +129,13 @@ def update_stale_cohort_metrics() -> None:
 
     COHORT_STUCK_COUNT_GAUGE.set(stuck_count)
 
+    maxed_error_count = (
+        Cohort.objects.filter(deleted=False, errors_calculating__gt=MAX_ERRORS_CALCULATING)
```

Comment:
Just confirming, should this be `errors_calculating__gte` because we stop trying when they hit 20 so we'd never have 21?

Reply:
We try cohorts if they have <=20 errors (see line 69 in this file `errors_calculating__lte=MAX_ERRORS_CALCULATING`). So 21 is the max automatic calculations can increment to.